### PR TITLE
emit in batches of 1000

### PR DESCRIPTION
**Pull request by corylanou**
@pauldix is this what you were thinking?

### CODE REVIEW

**Comment by pauldix** [comment]
yep, that's exactly it.

File: influxql/engine.go [hunk 1]
```diff
@@ -752,6 +752,11 @@ func MapStddev(itr Iterator, e *Emitter, tmax int64) {
 
 	for k, v := itr.Next(); k != 0; k, v = itr.Next() {
 		values = append(values, v.(float64))
+		// Emit in batches of 1000
```
Comment (otoolep) [inline influxql/engine.go]: If we're not going to make this configurable, perhaps we should at least add a comment explaining *why* we are doing this.

Comment (pauldix) [inline influxql/engine.go]: yeah good call. Pull the magic number out to a constant at least.

**Comment by otoolep** [comment]
This change still doesn't still me *why* we batch. @pauldix identified a reason, a good one I'm sure, but this code doesn't tell me what it is. It would be really helpful to the next person when reviewing this code, to understand why our system works like this.

**Comment by otoolep** [comment]
For example "unbounded emission of data can cause the system to run out of RAM", or whatever the reason is.

**Pull request merged**